karrot/activities/templates/activity_notification.mjml

```diff
@@ -19,12 +19,14 @@
                                         <a href="{{ place_url(activity.place) }}">
                                             {{ activity.place.name }}
                                         </a>
+                                        {# TODO maybe show if it requires a role or not... #}
                                         {% if show_empty_participants_count %}
                                             -
-                                            {% trans trimmed empty_participants_count=activity.empty_participants_count(),
+                                            {% trans trimmed empty_participants_count=activity.empty_participants_count(role=activity.require_role),
                                                     max_participants=activity.max_participants %}
                                                 {{ empty_participants_count }}/{{ max_participants }} remaining
                                             {% endtrans %}
+                                            {# TODO maybe show the available slots without role remaining too #}
```

Comment:
These TODOs would be useful to do!

Reply:
Well, this is a little bigger now, as we should take into account the roles of the user in which ones we show...